daemon/daemon.go

```diff
@@ -916,6 +917,59 @@ func createPrefixLengthCounter() *counter.PrefixLengthCounter {
 	return counter
 }
 
+func (d *Daemon) prepareAllocationCIDR(family datapath.NodeAddressingFamily) (routerIP net.IP, err error) {
```

Comment:
Please avoid running this if agent is being executed with flannel mode.

Reply:
Sure, can do but it was run before in flannel mode.